source/inference.md

```diff
@@ -1468,64 +1170,37 @@ To do this in Python, we can use the `percentile()` function from the `numpy` pa
 ```
 
 ```{code-cell} ipython3
-import numpy as np
-bounds = np.percentile(boot20000_means["mean"], [2.5, 97.5])
-
-bounds
+ci_bounds = boot20000_means["sample_mean"].quantile([0.025, 0.975])
```

Comment:
The text above this needs to be updated to be about the `quantile` method instead of percentile, and we should probably also point out the naming difference